index.bs

```diff
@@ -924,8 +923,14 @@ The <dfn>context type</dfn> is the type of the execution context that manages th
             : {{SharedArrayBuffer}}
             :: Return true.
             : {{ArrayBufferView}}
-            :: If |bufferSource|'s [=element type=] matches |descriptor|'s {{MLOperandDescriptor/dataType}} according to [this table](#appendices-mloperanddatatype-arraybufferview-compatibility) return true, otherwise return false.
+            ::
+                1. If |bufferSource| is a {{Uint8Array}} object, then return true.
+                1. If |bufferSource| matches |descriptor|'s {{MLOperandDescriptor/dataType}} according to [this table](#appendices-mloperanddatatype-arraybufferview-compatibility), then return true.
+                1. Return false.
         </dl>
+
+    Note: Using {{Uint8Array}} regardless of the |descriptor|'s {{MLOperandDescriptor/dataType}} is supported as a generic way of representing a slice of an {{ArrayBuffer}}, for example part of a {{WebAssembly}}.{{Memory}} instance. Developers are encouraged to use more specific view types when authoring WebNN code for readability and maintainability.
+
 </details>
```

Comment:
Is Float16Array at stage 4 now? If so, should we update this text too:

> Float16Array is at ECMA Stage 3 signaling its design is finished. Implementers wanting to enable this type ahead native implementations can emulate the type by passing raw bits via Uint16Array. [Issue webnn#373]

Reply:
Never mind - it still has 3 checkboxes for stage 4 completion: https://github.com/tc39/proposal-float16array/issues/7